src/server_manager/electron_app/index.ts

```diff
@@ -149,21 +149,21 @@ function workaroundDigitalOceanApiCors() {
   const headersFilter = {urls: ['https://api.digitalocean.com/*']};
   electron.session.defaultSession.webRequest.onHeadersReceived(
       // tslint:disable-next-line:no-any
-      headersFilter, (details: any, callback: Function) => {
+      headersFilter, (details: electron.OnHeadersReceivedListenerDetails, callback: (response: any) => void) => {
+        console.error(details);
```

Comment:
Is this debug logging that should be removed?

Reply:
Oops. Removed.